cmake/std/atdm/README.md

```diff
@@ -1162,6 +1163,89 @@ $ ./checkin-test-atdm.sh <buildname0> <buildname1> ... \
   use caution when doing so as not to disturb the job running.
 
 
+### ASTRA (Vanguard ARM System)
+
+Once logged onto a supported Vanguard ARM system (called system 'van1-tx2')
+system like 'stria', one can build and configure on a login node.
+
+To configure, build and run the tests for the default `arm-20.0` build for
+`Kokkos` (after cloning Trilinos on the 'develop' branch), run the following
+from a login node on 'stria':
+
+```bash
+$ cd <some_build_dir>/
+
+# List available environments
+$ source $TRILINOS_DIR/cmake/std/atdm/load-env.sh help
+
+# Load hsw env and configure on the login node
+$ source $TRILINOS_DIR/cmake/std/atdm/load-env.sh arm-20.0
+$ cmake -G Ninja \
+  -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/atdm/ATDMDevEnv.cmake \
+  -DTrilinos_ENABLE_TESTS=ON \
+  -DTrilinos_ENABLE_Tpetra=ON \
+  $TRILINOS_DIR
+
+$ make NP=8  # This is a shared node!
+
+# Get a node allocation and run ctest
+$ salloc -N 1 --time=2:00:00  -p short,batch --account=<YOUR_WCID> ctest -j4
+```
+
+One can also get an allocation first and then configure, build on a compute
+node, and then run the test suite using:
+
+```bash
+$ salloc -N 1 --time=4:00:00 -p short,batch --account=<YOUR_WCID> bash
+# NOTE: After the above runs, hostname=stria-login<n> but now a compute node
+# has been allocated for immediately usage.
+
+$ source $TRILINOS_DIR/cmake/std/atdm/load-env.sh arm-20.0
+
+$ cmake -G Ninja \
+  -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/atdm/ATDMDevEnv.cmake \
+  -DTrilinos_ENABLE_TESTS=ON \
+  -DTrilinos_ENABLE_Tpetra=ON \
+  $TRILINOS_DIR
+
+$ srun -N 1 make NP=20  # We have the entire compute node to ourselves!
+
+$ ctest -j4
+```
+
+The advantage of the latter approach is that one just waits once for a node
+allocationand then one can immediately run fast parallel builds on the compute
```

Comment:
:+1: This documentation looks great! Small typo here: "allocationand" -> "allocation and"